yoti-sdk-api/src/main/java/com/yoti/api/client/docs/DocScanService.java

```diff
@@ -386,6 +392,32 @@ public Media getIbvInstructionsPdf(String sdkId, KeyPair keyPair, String session
         }
     }
 
+    public SessionConfigurationResponse fetchSessionConfiguration(String sdkId, KeyPair keyPair, String sessionId) throws DocScanException {
+        notNullOrEmpty(sdkId, "SDK ID");
```

Comment:
Another class with unused imports

Is there scope for some reuse in this beast?